package.json

```diff
@@ -62,7 +62,15 @@
     "workbox-precaching": "^6.5.4",
     "workbox-routing": "^6.5.4",
     "workbox-strategies": "^6.5.4",
-    "workbox-window": "^6.5.4"
+    "workbox-window": "^6.5.4",
```

Comment:
Please recehck if you are using all the included ones.

Reply:
I don't understand. I'm not using workbox. Is it used for testing?

Reply:
I found a bunch of dependencies added as part of the PR. My question was, are you using all of them for your play?